geomstats/visualization.py

```diff
@@ -542,3 +559,89 @@ def plot(points, ax=None, space=None,
         klein_disk.draw(ax, **point_draw_kwargs)
 
     return ax
+
+
+class DataArrow():
```

Comment:
Should this be called `Arrow2D`, to be consistent with the class on line L42?

Or we could create a class `Arrow` whose constructor takes `dim` as a parameter?

Reply:
I feel this `DataArrow` class is quite specific to the example function, perhaps it's better to isolate it semantically speaking.
Moreover it seems mysteriously bugged.

Reply:
Ok! (Which one is bugged?)

Reply:
Mine =). It's not a critical bug, sometimes the arrows don't point to the correct spot in the figure.